Source/Modules/cffi.cxx

```diff
@@ -134,15 +134,15 @@ int CFFI::top(Node *n) {
   File *f_lisp = NewFile(lisp_filename, "w", SWIG_output_files());
   if (!f_lisp) {
     FileErrorDisplay(lisp_filename);
-    SWIG_exit(EXIT_FAILURE);
+    exit(EXIT_FAILURE);
```

Comment:
This should be `Exit(EXIT_FAILURE);` now (NB capitalisation exactly like this) - this provides an exit hooking mechanism as part of the DOH layer so we can remove any output files SWIG has started to create if it exits due to an error, which only happened in some cases before with `SWIG_exit()` - see 55377bdc08ffbe37fbb89566a8754302ae186227 if you want the gory details.